### PR TITLE
lncli: short command for invoice payment

### PR DESCRIPTION
**Pull request by rajeshnair2k**
lncli: Short command for invoice payment

### CODE REVIEW

File: cmd/lncli/commands.go [hunk 1]
```diff
@@ -881,6 +885,40 @@ func sendPayment(ctx *cli.Context) error {
...
 }
 
+var payInvoiceCommand = cli.Command{
+	Name:      "payinvoice",
+	Usage:     "pay an invoice over lightning",
+	ArgsUsage: "pay_req",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "pay_req",
+			Usage: "a zbase32-check encoded payment request to fulfill",
```
Comment (halseth) [inline cmd/lncli/commands.go]: this should be changed to zpay32-encoded here and elsewhere.

Comment (halseth) [reply cmd/lncli/commands.go]: I'm also not sure you really need the `pay_req` flag, as this command takes only one argument. Instead you could add a `Description` field.

Comment (rajeshnair2k) [reply cmd/lncli/commands.go]: @halseth I am happy to do that. But I added that to keep it consistent with other single argument commands. decodePayReqComamnd is an example.

Comment (halseth) [reply cmd/lncli/commands.go]: Ok, makes sense. Guess it doesn't hurt to have it as an option :)

**Comment by rajeshnair2k** [comment]
@halseth I have made the requested changes and rebased on master.

**Comment by rajeshnair2k** [comment]
@halseth let me know if you got a chance to test. Also let me know if I need to make any other changes.

File: cmd/lncli/main.go [hunk 2]
```diff
@@ -177,6 +177,7 @@ func main() {
 		verifyMessageCommand,
 		feeReportCommand,
 		updateFeesCommand,
+		payInvoiceCommand,
```
Comment (halseth) [inline cmd/lncli/main.go]: The placement of the payinvoice command probably should be right above (or below?) the sendpayment command in the list.

Comment (rajeshnair2k) [reply cmd/lncli/main.go]: @halseth I have moved the command below sendPayment along with other invoice commands. PTAL :).

**Comment by rajeshnair2k** [comment]
@halseth I have moved the command below sendPayment along with other invoice commands. Let me know if there's anything else I need to do to get this merged. Thanks for your help :).

**Comment by halseth** [comment]
@rajeshnair2k Looks good in my opinion now 😀

**Comment by rajeshnair2k** [comment]
@roasbeef let me know if you get a chance to look at this.

**Pull request merged**